src/client/app/main.jsx

```diff
@@ -19,15 +20,45 @@ import '../styles/styles.scss';
 
 let networkInterface = createBatchingNetworkInterface({
   opts: {
-    credentials: "same-origin",
+    credentials: "include",
   },
   batchInterval: 20,
   uri: __BACKEND_URL__ || "/graphql",
 });
 if (__CLIENT__) {
+  networkInterface.use([{
+    applyBatchMiddleware(req, next) {
+      if (!req.options.headers) {
+        req.options.headers = {};
+      }
+
+      req.options.headers['x-token'] = window.localStorage.getItem('token');
+      req.options.headers['x-refresh-token'] = window.localStorage.getItem('refreshToken');
```

Comment:
Is there a way to abstract this into the user-module?

Reply:
will take a look

Reply:
Something like this? https://github.com/sysgears/apollo-universal-starter-kit/pull/338
Still working on connectionParams

Reply:
should be done